src/styles/todoapp.scss

```diff
@@ -130,4 +130,8 @@
       text-decoration: none;
     }
   }
+
+  &__clearbutton-replacer {
+    margin-left: 20%;
```

Comment:
Consider using a more descriptive class name for the clear button style instead of '__clearbutton-replacer'.